src/lib/AuthzUrlBuilder.ts

```diff
@@ -80,7 +101,7 @@ export class AuthzUrlBuilder {
     return this
   }
 
-  prompt(params: string) {
+  prompt(params?: IPromptType) {
```

Comment:
Here we need a mandatory param because it's meaning less to have an optional param in the Builder. If the user doesn't want the `prompt` param, he just don't call this method.

Reply:
done

Reply:
> Here we need a mandatory param because it's meaningless to have an optional param in the Builder. If the user doesn't want the `prompt` param, he just doesn't call this method.

done